pramen/core/src/main/scala/za/co/absa/pramen/core/utils/JdbcNativeUtils.scala

```diff
@@ -137,6 +146,10 @@ object JdbcNativeUtils {
         throw ex
     }
 
+    jdbcConfig.fetchSize.foreach(n =>
+      statement.setFetchSize(n)
+    )
```

Comment:
This is the third fix for PostgreSQL. The fetch size should be set _explicitly_. Useful for other JDBC sources as well I think.